random/module.py

```diff
@@ -35,6 +35,9 @@ async def pick(self, ctx, first: str, second: str, *args):
                 args = args[i + 1:]
                 break
 
+        if len(args) < 2:
+            return
```

Comment:
Can you add some reason why the bot did not pick any item from the list as `.reply()`?
```py
await ctx.reply(_(ctx, "You asked a question, but did not add any options."))
return
```
or something similar, the specific message does not matter that much.

Reply:
(The underscore function is used for translation, you don'ŧ have to deal with that.)